js/script.js

```diff
@@ -1,18 +1,84 @@
+// Mobile Menu Variables
 const containerMenu = document.querySelector('.menu');
 const closeMenu = document.querySelector('.menu-close');
 const openMenu = document.querySelector('.menu-open');
 const linkMenu = document.querySelectorAll('.menu a');
 
+// Modal Variables
+const containerModal = document.querySelector('.modal');
+const openModal = document.querySelectorAll('.modal-open');
+const infoModal = {
+  title: 'Project name goes here',
+  language: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+  img: 'assets/img/Project2.png',
+  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi <br> <br>
+  Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.`,
+  liveUrl: '#',
+  seeUrl: '#',
+};
```

Comment:
- [ ] The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
- Try to push some elements in the variable `infoModal` (I think 4 would be great) and for each "see this project" button show the info for each element. for example:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
-This will help you to make it easy the page with diff projects.

Reply:
This change has not yet been implemented propery.